flipperdevices/flipperzero-firmware
# [FL-3123] Reset sector cache when the SD card isn't working

# What's new

- The SD card cache gets reset on every error

# Verification 

- Use the Flipper as you would normally, check for any abnormal behavior
- Remove/swap/reconnect the SD card in the process of using the Flipper and check that it doesn't slow down or crash

# Checklist (For Reviewer)
